.rubocop.yml

```diff
@@ -0,0 +1,30 @@
+# The behavior of RuboCop can be controlled via the .rubocop.yml
+# configuration file. It makes it possible to enable/disable
+# certain cops (checks) and to alter their behavior if they accept
+# any parameters. The file can be placed either in your home
+# directory or in some project directory.
+#
+# RuboCop will start looking for the configuration file in the directory
+# where the inspected file is and continue its way up to the root directory.
+#
+# See https://docs.rubocop.org/rubocop/configuration
+require:
+  - rubocop-on-rbs
+
+AllCops:
+  TargetRubyVersion: 3.3
+  NewCops: disable
+  DisabledByDefault: true
+  Exclude:
+    - "**/Steepfile"
+    - "**/*.rb"
+    - "**/vendor/**/*"
+
+RBS:
+  Enabled: true
+RBS/Layout:
+  Enabled: false
+RBS/Lint:
+  Enabled: false
+RBS/Style:
+  Enabled: false
```

Comment:
Let me confirm this configuration. Is this configuration for the `RBS` department necessary?
I guess we can simply remove these lines because `DisabledByDefault: true` is configured.

Reply:
It was necessary to actually run it.

RBS/Layout is not enabled with the following settings Perhaps `DisabledByDefault: true` is in effect.

```yml
inherit_from: ../../.rubocop.yml

RBS/Layout:
  Enabled: true
```

The following settings enable `RBS/Layout`, but also enable `RBS/Lint` and `RBS/Style`.

```yml
inherit_from: ../../.rubocop.yml

RBS:
  Enabled: true
RBS/Layout:
  Enabled: true
```

Reply:
I understand the problem. When `DisabledByDefault` is true, RuboCop does not enable cops under `RBS/Lint` even if the configuration enables `RBS/Lint`. But we need to enable the `RBS` department explicitly.
It sounds a bug of RuboCop, but I think it's ok to configure it in our configuration file 👍